source/stylesheets/_tab-workflows.scss.erb

```diff
@@ -494,6 +494,8 @@ section.manage-workflows {
 									background-color: $def-danger;
 									@include transition(background-color $def-transition_duration);
 
+									box-shadow: 0px 0px 0px 2px white;
```

Comment:
No variable for the bg color? It's fine by me, but if there is one, we could use that.

Reply:
No variable I checked and it was just 'white'.